coopertunes/models/Audio2Mel.py

```diff
@@ -0,0 +1,45 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import torch
+from librosa.filters import mel as librosa_mel_fn
+
+from coopertunes.hparams import HParams
+
+
+class Audio2Mel(nn.Module):
```

Comment:
O wow, why we need a `nn.Module` for that. if you need mels there is also func for it in utils

Reply:
If you need differentiable mels there is still `torchaudio.transforms.MelSpectrogram`

Reply:
Sure, this one is for processing audio online if there is a need (explanation in some other comment here above). Generally as we implement it as module, we can run it in series with trained model on GPU.